projects/plugins/jetpack/extensions/blocks/sharing-button/edit.js

```diff
@@ -21,10 +21,17 @@ const SharingButtonEdit = ( { attributes, context } ) => {
 		className: 'jetpack-sharing-button__list-item',
 	} );
 
+	const defaultIconColor = styleType === 'icon' ? '#fff' : '#000';
+
+	const buttonStype = {
```

Comment:
Minor typo, I believe you meant `buttonStyle` 😃

Reply:
Eagle eye 🦅